concepts/05 UI Components/PivotGrid/00 Getting Started with PivotGrid/16 Configure the Field Chooser.md

```diff
@@ -0,0 +1,125 @@
+The field chooser allows users to manage pivot grid fields. Its window displays five field sections, four of which correspond to pivot grid areas and the fifth contains all fields. Users can drag and drop fields between these sections.
+
+To open the field chooser, users should click the Field Chooser icon or select Show Field Chooser from a row or column header's context menu.
+
+![DevExtreme PivotGrid: Field chooser](/images/pivotgrid/field-chooser.png)
+
+To configure the field chooser, declare the [fieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldChooser/) object. Use the [enabled](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldChooser/#enabled) property to specify whether the Field Chooser icon and the corresponding context menu command are available. This property's default value is **true**.
+
+The All Fields section includes fields declared in the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) array and fields auto-generated from the data source model. If the All Fields section contains so many fields that users find it hard to navigate them, you can disable the [dataSource](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#dataSource).[retrieveFields](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/#retrieveFields) property to hide the auto-generated fields. Alternatively, you can enable the [allowSearch](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldChooser/#allowSearch) property to allow users to search in the All Fields section.
```

Comment:
> The All Fields section includes **fields** declared in the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) array and **fields** auto-generated from the data source model.

In both instances, the "fields" have grammatical definiteness due to the clarification you gave to each. **Those/all** fields declared in ... and **those/all** fields that were automatically generated ...

```suggestion
The All Fields section includes the fields declared in the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) array and the fields auto-generated from the data source model. If the All Fields section contains so many fields that users find it hard to navigate them, you can disable the [dataSource](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#dataSource).[retrieveFields](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/#retrieveFields) property to hide the auto-generated fields. Alternatively, you can enable the [allowSearch](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldChooser/#allowSearch) property to allow users to search in the All Fields section.
```